### PR TITLE
Fix skeleton not showing when no filter has been selected for AB#13464.

### PR DESCRIPTION
**Pull request by BrianMaki**
# Fixes [AB#13464](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/13464)

## Description

This fix addresses the issue when user clicks on the Health Records card after doing a refresh on the home page and the user is directed to the timeline, the skeleton is not appearing.

## Testing

- [ ] Unit Tests Updated
- [ ] Functional Tests Updated
- [x] Not Required

## UI Changes

No

## Items to Review:

-   [General PR Guidelines](https://github.com/bcgov/healthgateway/wiki/PRguidance)

### CODE REVIEW

**Review by rayking29 — DISMISSED** [review]

File: Apps/WebClient/src/ClientApp/src/views/TimelineView.vue [hunk 1]
```diff
@@ -344,10 +344,19 @@ export default class TimelineView extends Vue {
...
+    }
+
     private get showContentPlaceholders(): boolean {
-        const showContent = this.isFilterLoading;
-        this.logger.debug(`Timeline show content placeholders: ${showContent}`);
-        return showContent;
+        if (this.isFilterModuleSelected) {
+            return this.isFilterLoading;
+        }
+        return !this.isFullyLoaded;
```
[the remark this answers is not in the document]
Comment (MikeLyttle) [reply Apps/WebClient/src/ClientApp/src/views/TimelineView.vue:359]: And maybe `this.filteredEntries.length === 0` should also be applied when module(s) are selected.

[the remark this answers is not in the document]
Comment (BrianMaki) [reply Apps/WebClient/src/ClientApp/src/views/TimelineView.vue:359]: > filteredEntries

There is no this.filteredEntries.length. Are you referring to filteredTimelineEntries.length?

[the remark this answers is not in the document]
Comment (MikeLyttle) [reply Apps/WebClient/src/ClientApp/src/views/TimelineView.vue:359]: Yes, that's the one.

[the remark this answers is not in the document]
Comment (BrianMaki) [reply Apps/WebClient/src/ClientApp/src/views/TimelineView.vue:359]: I added that in.  Both implementation work.  When filter is not being used (clicking on Health Records Card) we want to display the 'Content Placeholders' when 'isFullyLoaded' is false i.e., as long as one API returns.  So I am believing return !this.isFullyLoaded should be enough.  No?

[the remark this answers is not in the document]
Comment (MikeLyttle) [reply Apps/WebClient/src/ClientApp/src/views/TimelineView.vue:359]: I believe we only want the placeholders to display when no records are present and data sets are still loading.  If one data set returns with records, but the others are still loading, it won't be fully loaded, but the placeholders should go away.  @ninosamson should confirm.

[the remark this answers is not in the document]
Comment (BrianMaki) [reply Apps/WebClient/src/ClientApp/src/views/TimelineView.vue:359]: > I believe we only want the placeholders to display when no records are present and data sets are still loading. If one data set returns with records, but the others are still loading, it won't be fully loaded, but the placeholders should go away. @ninosamson should confirm.

I believe it does do that.  Check mock environment.  Click on Notes Card on home.

[the remark this answers is not in the document]
Comment (BrianMaki) [reply Apps/WebClient/src/ClientApp/src/views/TimelineView.vue]: I will make the change.

**Pull request merged**